features/wallet/impl/src/main/java/com/tangem/feature/wallet/presentation/wallet/viewmodels/WalletViewModel.kt

```diff
@@ -74,8 +76,11 @@ internal class WalletViewModel @Inject constructor(
     /** Feature router */
     var router: InnerWalletRouter by Delegates.notNull()
 
+    /** Screen state */
+    val uiState: WalletState get() = uiStateHolder.uiState
```

Comment:
Можно чуть упростить:
```suggestion
    var uiState: WalletState get() = uiStateHolder.uiState
        private set(value) {
            uiStateHolder.setState(value)
        }
```

Reply:
Добавил делагат для этого дела